templates/index.pug

```diff
@@ -52,4 +54,4 @@ html
     )
   body
     #app
-    script(src='/base.bundle.js')
+    script(src='./base.bundle.js')
```

Comment:
This should be kept absolute since the javascript file will always be at the root.

Reply:
Cool I changed this back in my latest push. Have you thought about adding Browser-sync as a devDependency?